### ISSUE TITLE
[BUG] Accessibility/VoiceOver: Not intractable text on the notifications overview will be called out as buttons

### ISSUE THREAD

**Issue by ericyahhh** [mastodon/mastodon-ios#541]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

- Go to the Notifications screen
- Turn on VoiceOver
- Swipe right to go to the content

**Issue 1**: Small "dot"will be announced and called out as a button

![IMG_9044](https://user-images.githubusercontent.com/1116348/200835124-87ab1bef-f96f-4dfe-89cb-d06c45a7cc5f.jpg)

**Issue 2**: Timestamp will be called out as a button

![IMG_9045](https://user-images.githubusercontent.com/1116348/200835114-8b78f789-fcfa-4af8-9ebe-14e50813087d.jpg)

### Expected Behavior

Issue 1:
- The small "dot" shouldn't be announced via VoiceOver at all

Issue 2:
- The timestamp should be announced, but not called out as a button

### Steps To Reproduce

See above

### Environment

```markdown
- Device: iPhone 12 mini
- OS: iOS 16.1
- Version: 1.4.5
- Build: 144
```

**Comment by j-f1** [comment]
Looking into notification accessibility in general now!

**Comment by j-f1** [comment]
Fixed along with several other issues in #552!
